src/components/input-dropdown/input-dropdown.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="ux-select-container">
+    <button #button type="button" class="form-control"
+            [uxMenuTriggerFor]="menu">
+        <ng-template #defaultDisplayContent>{{selectedString()}}</ng-template>
+        <ng-container [ngTemplateOutlet]="displayContentRef ? displayContentRef : defaultDisplayContent"></ng-container>
+    </button>
+    <div class="ux-select-icons">
+        <ux-icon name="close"
+                 class="ux-select-icon ux-select-clear-icon"
+                 *ngIf="allowNull && selected"
+                 (click)="resetValue($event)">
+        </ux-icon>
+        <ux-icon name="chevron-down"
+                 class="ux-select-icon ux-select-chevron-icon"
+                 (click)="menuTrigger.toggleMenu(); $event.stopPropagation()">
+        </ux-icon>
+    </div>
+</div>
+
+<ux-menu #menu menuClass="select-menu"
+         (opened)="filterInputElement.nativeElement.focus()">
+
+    <div [style.max-height.px]="maxHeight"
```

Comment:
Instead of forcing this to be a pixel value, we should allow any CSS value here. We can use the setter and the Angular CDK coercion utilities to default to pixels if no unit is provided (https://github.com/angular/components/blob/6.0.x/src/cdk/coercion/css-pixel-value.ts)

Something like this:
```typescript
set maxHeight(value: string | any) {
    this._maxHeight = coerceCssPixelValue(value);
}
```

Reply:
I changed that. Setting a pure number does not work on the demo page, though, but when I set it manually.